### PR TITLE
#19: Add project-keeper

### LINKED ISSUE #19: Add project-keeper
Version 0.7.3 of prject-keeper fixed an issued that made it impossioble to use it here

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #19

### CODE REVIEW

File: doc/changes/changes_1.0.1.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ Code name: API fix
 
 In this release we added missing methods to the Row interface.
 
+## Refactoring
+
+* #19: Add project-keeper
```
Comment (morazow) [inline doc/changes/changes_1.0.1.md:11]:
```suggestion
* #19: Added project-keeper
```

File: dependencies.md [hunk 2]
```diff
@@ -27,66 +27,68 @@
...
 [0]: https://parquet.apache.org
 [2]: https://www.apache.org/licenses/LICENSE-2.0.txt
-[23]: https://github.com/scalatest/scalatest-maven-plugin
+[23]: http://nexus.sonatype.org/oss-repository-hosting.html/scalatest-maven-plugin
 [33]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
 [10]: https://github.com/mockito/mockito/blob/release/3.x/LICENSE
 [8]: https://www.eclipse.org/legal/epl-v20.html
-[3]: https://github.com/google/guava
-[59]: http://maven.apache.org/plugins/maven-install-plugin/
+[3]: https://github.com/google/guava/guava
```
Comment (morazow) [inline dependencies.md:84]: I think this fails with link checker. This Guava link is somehow wrong from dependency.

File: pom.xml [hunk 3]
```diff
@@ -497,6 +507,40 @@
...
+                    </execution>
+                </executions>
+                <configuration>
+                    <modules>
+                        <module>maven_central</module>
+                        <module>integration_tests</module>
+                    </modules>
+                    <excludedPlugins>
+                        <!-- does not work with scalar code yet -->
+                        <excludedPlugin>com.exasol:error-code-crawler</excludedPlugin>
```
Comment (morazow) [inline pom.xml:528]: :+1:

**Review by morazow — APPROVED** [review]
This is nice! 
Added minor suggestions, overall looks good.

File: pom.xml [hunk 4]
```diff
@@ -480,7 +491,6 @@
                     <serialwarn>true</serialwarn>
                     <failOnError>true</failOnError>
                     <failOnWarnings>true</failOnWarnings>
-                    <sourcepath>${project.basedir}/src/main/java</sourcepath>
```
Comment (morazow) [inline pom.xml:483]: This also causes Javadoc to fail.

File: dependencies.md (hunk 2 above)
Comment (jakobbraun) [reply dependencies.md:84]: Yea, I created a PR: https://github.com/google/guava/pull/5619

**Pull request merged**